RATIONALE.md

```diff
@@ -0,0 +1,32 @@
+# zipkin-storage-kafka rationale
+
+## Use-cases
+
+### Replacement for Spark-based Zipkin dependencies
+
+One limitation of [zipkin-dependencies](https://github.com/openzipkin/zipkin-dependencies) module based on Spark is that it requires to be scheduled with a defined frequency. This batch-oriented execution causes out-of-date values until processing runs again.
+
+A Kafka-based storage brings the opportunity to aggregate dependencies as spans are received, allowing a (near-)real-time calculation of dependency metrics.
+
+To enable this, other components could be disabled:
+
+```yaml
```

Comment:
how about package a yaml profile instead?

Reply:
I'd assume this will be similar to slim profile right? I'll work on it :)

Reply:
at first, just a differnt yaml file should be ok (with the options set). I don't think you need a slim build, just another file which is used with the spring args

Reply:
another option is an uber flag which defaults all the things..